src/components/LabListEditor/LabListEditor.tsx

```diff
@@ -27,12 +31,24 @@ export function LabListEditor({labs, prefix, moduleId}: Props) {
           <Row style={{marginBottom: '1rem'}}>
             <Col>Labs</Col>
             <Col className='d-flex justify-content-end align-items-center'>
-              <IconButton
-                icon={faPlusCircle}
-                size={'2x'}
-                link={getNewLabEditorLink(moduleId)}
-                color={'black'}
-              />
+              { disabledModule ?
+                <>
+                  <ReactTooltip place='left' type='dark' effect='solid'/>
+                  <span data-tip='"Disable Module" must be unchecked to add a Lab'>
+                    <IconButton
+                      icon={faPlusCircle}
+                      size={'2x'}
+                      color={'grey'}
+                    />
+                  </span>
+                </>
+              :
+                <IconButton
+                  icon={faPlusCircle}
+                  size={'2x'}
+                  link={getNewLabEditorLink(moduleId)}
+                  color={'black'}
+                />}
```

Comment:
Move the React tooltip into the IconButton, adding it as an extra layer on top causes extra whitespace to be added accidentally.

Then you can combine the turnery into one component call with smaller tuneries
```suggestion
                  <IconButton
                    icon={faPlusCircle}
                    size={'2x'}
                    tooltip={ disabledModule  ? '"Disable Module" must be unchecked to add a Lab' : undefined}
                    color={disabledModule   ? 'grey' : 'black'}
                    link={!disabledModule   ? getNewLabEditorLink(moduleId) : undefined}
                  />
```